Subject: mlx5: Adjust stride num and stride size values.

Adjust the number of strides to what kernel expects and adjust
the stride size for performance gains in case of smaller
packets size.

Signed-off-by: Sameer Agrawal <sa503s@att.com>
--- a/drivers/net/mlx5/mlx5_defs.h
+++ b/drivers/net/mlx5/mlx5_defs.h
@@ -144,10 +144,10 @@
 #endif
 
 /* Log 2 of the default number of strides per WQE for Multi-Packet RQ. */
-#define MLX5_MPRQ_STRIDE_NUM_N 6U
+#define MLX5_MPRQ_STRIDE_NUM_N 9U
 
 /* Log 2 of the default size of a stride per WQE for Multi-Packet RQ. */
-#define MLX5_MPRQ_STRIDE_SIZE_N 11U
+#define MLX5_MPRQ_STRIDE_SIZE_N 8U
 
 /* Two-byte shift is disabled for Multi-Packet RQ. */
 #define MLX5_MPRQ_TWO_BYTE_SHIFT 0
